roles/build_openstack_packages/tasks/run_dlrn.yml

```diff
@@ -14,15 +14,25 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Set proper branch name
+  ansible.builtin.set_fact:
+    _project_branch: "{{ cifmw_bop_openstack_release }}"
+  when: _change.branch != cifmw_bop_openstack_release
+
+- name: Set proper branch name
```

Comment:
Do we need this task and the one above? It looks like we always end up using the value of `cifmw_bop_openstack_release`.

Reply:
that said, it could be done in a follow up, if this is blocking something

Reply:
We need both the tasks. Let me change the wording of the task to make it clearer.